products/opensuse/main.pm

```diff
@@ -758,6 +758,10 @@ sub load_slenkins_tests {
 if (is_kernel_test()) {
     load_kernel_tests();
 }
+elsif (get_var("NETWORKD")) {
+    boot_hdd_image();
+    load_networkd_tests();
+}
```

Comment:
could you please check if it also works for SLE?

At best you can move this into `load_common_opensuse_sle_tests`

Reply:
I think this should then be changed for the wicked tests as well and should be done combined in a separate PR.